guides/doc-Configuring_User_Authentication/master.adoc

```diff
@@ -0,0 +1,65 @@
+include::common/attributes.adoc[]
+include::common/header.adoc[]
+:context: authentication
+
+= {ConfiguringUserAuthenticationDocTitle}
+
+// This guide is not ready for stable releases
+ifdef::HideDocumentOnStable[]
+include::common/modules/snip_guide-not-ready.adoc[]
+endif::[]
+ifndef::HideDocumentOnStable[]
+
+ifdef::satellite[]
+include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[leveloffset=+1]
+endif::[]
+
+include::common/assembly_accessing-server.adoc[leveloffset=+1]
+
+include::common/assembly_configuring-an-ldap-server-as-an-external-identity-provider-for-project.adoc[leveloffset=+1]
+
+include::common/assembly_configuring-kerberos-single-sign-on-with-freeipa-in-project.adoc[leveloffset=+1]
+
+// The following ifdef sets :keycloak: to `Red Hat Single Sign-On` for Satellite builds only. For, non-Satellite builds, :keycloak: stays set to `Keycloak`.
+ifdef::satellite[]
+:keycloak: {keycloak-wildfly}
+:keycloak-example-com: rhsso.example.com
+endif::[]
+ifdef::context[:parent-context: {context}]
```

Comment:
```suggestion
parent-context: {context}
```

We always have "context" set to something, i.e. "authentication" in this guide.

Same suggestion applies to lines 31,32,44,47,48.

Reply:
Addressed. It does simplify things. And I added explanatory comments too.